wwwroot/inc/snmp.php

```diff
@@ -4095,6 +4142,12 @@
 		'text' => 'Juniper EX4200 series',
 		'processors' => array ('juniper-ex-pic0-1000T', 'juniper-ex-mgmt'),
 	),
+	'2636.1.1.1.2.41' => array
+    (
+    	'dict_key' => 3733,
+        'text' => 'Juniper SRX 100: 8 RJ-45/10-100TX + 1 USB (support for 3G USB)',
+        'processors' => array ('Juniper-100TX'),
```

Comment:
this processor doesn't exist.

for this and other cases, please adjust the `doSwitchSNMPmining` function to ensure that appropriate ports get added.  in this case, power & usb.